src/agoradatatools/etl/transform/model_details.py

```diff
@@ -96,7 +96,7 @@ def process_genetic_info(
     )
     return (
         merged_df[["gene", "ensembl_id", "allele", "allele_type", "mgi_allele_id"]]
-        .rename(columns={"gene": "modified_gene"})
+        .rename(columns={"gene": "modified_gene", "ensembl_id": "ensembl_gene_id"})
```

Comment:
Looking at this file, in the line just above this you do:
```
merged_df["ensembl_id"] = merged_df["human_ensembl_id"].fillna(
        merged_df["gene_ensembl_id"]
    )
```
Is "ensembl_id" a column you create in that line of code or was it in the data frame when it got ingested? If you create it, why not create it with the right name right away? If it was already in the data frame, you could use `column_rename` in the config file instead of renaming here.

Reply:
Looks like you could also use `column_rename` to rename the "gene" column in the config as well.